components/automate-deployment/pkg/server/server.go

```diff
@@ -1888,7 +1924,13 @@ func (s *server) IsValidUpgrade(ctx context.Context, req *api.UpgradeRequest) (*
 
 // Upgrade requests the deployment-service pulls down the latest manifest and applies it
 func (s *server) Upgrade(ctx context.Context, req *api.UpgradeRequest) (*api.UpgradeResponse, error) {
-	if !s.HasConfiguredDeployment() {
+
+	validatedResp, err := s.IsValidUpgrade(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+
+	/*if !s.HasConfiguredDeployment() {
```

Comment:
can we get rid of  this commented block?

Reply:
Done